jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointCaller.java

```diff
@@ -140,8 +135,6 @@ static boolean isBrokenPipe(IOException original) {
       @Nullable Function<URL, Connection> insecureConnectionFactory)
       throws MalformedURLException {
     this.eventHandlers = eventHandlers;
-    this.initialRequestUrl =
-        registryEndpointProvider.getApiRoute(DEFAULT_PROTOCOL + "://" + apiRouteBase);
```

Comment:
Can we remove the `DEFAULT_PROTOCOL` constant, or is it used elsewhere? (even if it is, I think just hardcoding "https" is fine)

Reply:
Good catch.